base_layer/core/src/base_node/state_machine_service/states/events_and_states.rs

```diff
@@ -230,6 +240,13 @@ impl StateInfo {
             Listening(info) => info.is_synced(),
         }
     }
+
+    pub fn get_initial_connected_peers(&self) -> u64 {
+        match self {
+            StateInfo::Listening(info) => info.initial_delay(),
+            _ => 0,
+        }
+    }
```

Comment:
The naming here is really confusing; I would expect `get_initial_connected_peers` to return `initial_connected_peer`, not `initial_delay`.

Reply:
good point